xml/System.Net.Sockets/SocketType.xml

```diff
@@ -66,7 +66,7 @@
         <ReturnType>System.Net.Sockets.SocketType</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Supports datagrams, which are connectionless, unreliable messages of a fixed (typically small) maximum length. Messages might be lost or duplicated and might arrive out of order. A <see cref="T:System.Net.Sockets.Socket" /> of type <see cref="F:System.Net.Sockets.SocketType.Dgram" /> requires no connection prior to sending and receiving data, and can communicate with multiple peers. <see cref="F:System.Net.Sockets.SocketType.Dgram" /> uses the Datagram Protocol (<see cref="F:System.Net.Sockets.ProtocolType.Udp" />) and the <see cref="F:System.Net.Sockets.AddressFamily.InterNetwork" /><see cref="T:System.Net.Sockets.AddressFamily" />.</summary>
+        <summary>Supports datagrams, which are connectionless, unreliable messages of a fixed (typically small) maximum length. Messages might be lost or duplicated and might arrive out of order. A <see cref="T:System.Net.Sockets.Socket" /> of type <see cref="F:System.Net.Sockets.SocketType.Dgram" /> requires no connection prior to sending and receiving data, and can communicate with multiple peers. <see cref="F:System.Net.Sockets.SocketType.Dgram" /> uses the Datagram Protocol (<see cref="F:System.Net.Sockets.ProtocolType.Udp" />) and the <see cref="F:System.Net.Sockets.AddressFamily.InterNetwork" />.<see cref="T:System.Net.Sockets.AddressFamily" />.</summary>
```

Comment:
I think that this was a failed attempt to create a type/member name, @svick, so I suspect that the period is missing. At the same time, either adding a period or adding a space doesn't do much to improve this topic, which seems to me to have a certain lack of specificity. What I'd like to do, @tmpreston and @svick, is leave this PR open until I have a chance to revise the topic slightly. I'll then close this PR, open the new PR, and cc both of you to review the new PR.